backend/FileSystem/Path.php

```diff
@@ -4,26 +4,26 @@
 
 namespace Internships\FileSystem;
 
+define("PATHS_FOLDER_SEPARATOR", "/");
```

Comment:
Hmm. Global consts are not the great idea. Let's back to the public const in the `Path` class.